classes/class-wc-connect-nux.php

```diff
@@ -93,11 +102,44 @@ public function register_add_service_to_zone_pointer( $pointers ) {
 						__( 'To ship products to customers using USPS or Canada Post, you will need to add them as a shipping method to an applicable zone. If you don\'t have any zones, add one first.', 'woocommerce-services' )
 					),
 					'position' => array( 'edge' => 'right', 'align' => 'left' ),
-				)
+				),
 			);
 			return $pointers;
 		}
 
+		private function is_new_labels_user() {
+			$is_new_user = get_transient( self::TRANSIENT_IS_NEW_LABEL_USER );
+			if ( ! is_string( $is_new_user )) {
+				error_log( 'calculating if the user is new' );
+				global $wpdb;
+				$query = "SELECT meta_key FROM {$wpdb->postmeta} WHERE meta_key = 'wc_connect_labels' LIMIT 1";
+				$results = $wpdb->get_results( $query );
+				$is_new_user  = 0 === count( $results ) ? 'yes' : 'no';
+				set_transient( self::TRANSIENT_IS_NEW_LABEL_USER, $is_new_user );
+			}
+
+			return 'yes' === $is_new_user;
+		}
+
+		public function register_order_page_labels_pointer( $pointers ) {
+			if ( $this->is_new_labels_user() && $this->shipping_label->should_show_meta_box() ) {
+				$pointers[] = array(
+					'id' => 'wc_services_labels_metabox',
+					'target' => '#woocommerce-order-label',
+					'options' => array(
+						'content' => sprintf( '<h3>%s</h3><p>%s</p>',
+							__( 'Discounted Shipping Labels' ,'woocommerce-services' ),
+							__( 'When you\'re ready, purchase and print discounted labels from USPS right here.', 'woocommerce-services' )
```

Comment:
We can use double quote-marks here to [avoid escaping](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#single-and-double-quotes) using `\`.